packit/source_git.py

```diff
@@ -74,9 +74,16 @@ def __init__(
         self.source_git = source_git
         self.dist_git = dist_git
         self.upstream_ref = upstream_ref
-        self.upstream_url = upstream_url or next(
-            self.source_git.remote(upstream_remote or "origin").urls
-        )
+        try:
+            self.upstream_url = upstream_url or next(
+                self.source_git.remote(upstream_remote or "origin").urls
+            )
+        except ValueError:
```

Comment:
sorry, one more thing, I'd `logger.debug` the ValueError exception just in case it's something else and we have the information stored to ease debugging

otherwise LGTM, very nice work!

Reply:
Thank you, not a problem. Added logger.debug to except block.